### PR TITLE
Add yq script

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
This commit adds a bash script to test if YQ is installed and installs it if it isn't.

### CODE REVIEW

File: openshift-ci/scripts/oci-model-registry-deploy.sh [hunk 1]
```diff
@@ -157,6 +158,7 @@ run_deployment_tests() {
 
 # Main function for orchestrating deployments
 main() {   
+    ./install_yq.sh
```
Comment (tarilabs) [inline openshift-ci/scripts/oci-model-registry-deploy.sh]: at line 32 in `openshift-ci/scripts/install_yq.sh` you "Call the function when the script is sourced".

you source it at line 9.

didn't understand why you need this one here?

**Review by tarilabs — COMMENTED** [review]
minor but 

/lgtm

File: openshift-ci/scripts/oci-model-registry-deploy.sh (hunk 1 above)
Comment (tonyxrmdavidson) [reply openshift-ci/scripts/oci-model-registry-deploy.sh]: You are correct Matteo, changing it now.

Comment (tonyxrmdavidson) [reply openshift-ci/scripts/oci-model-registry-deploy.sh]: I think it might be easier to install yq via the openshift-ci deployment.

**Review by tarilabs — COMMENTED** [review]
/lgtm

**Pull request merged**